ram/wram.asm

```diff
@@ -2097,88 +2055,22 @@ wUnusedCardKeyGateID:: db
 
 	ds 8
 
-; bit 0: using Strength outside of battle
-; bit 1: set by IsSurfingAllowed when surfing's allowed, but the caller resets it after checking the result
-; bit 3: received Old Rod
-; bit 4: received Good Rod
-; bit 5: received Super Rod
-; bit 6: gave one of the Saffron guards a drink
-; bit 7: set by ItemUseCardKey, which is leftover code from a previous implementation of the Card Key
-wd728:: db
-
+wStatusFlags:: db
```

Comment:
I would probably prefer `wStatusFlags1` to avoid suggesting that these are _the_ status flags.

Reply:
how about `wOverworldFlags1` as the naming since status could be confused with for example...a pokemon having SLEEP status

Reply:
`wStatusFlags*` is another name similar to pokecrystal, and nobody has been that confused about it [yet].

Reply:
Could do something like wWorldStatus1 similar to wPlayerBattleStatus to differentiate